src/types/zodTypes.ts

```diff
@@ -240,3 +242,82 @@ export const DeliveryV1ResponseSchemaForOauth = z
     message: "authErrorCategory can't be empty when status is not a 2XX",
     path: ['authErrorCategory'], // Pointing out which field is invalid
   });
+
+export const ProcessorStreamingResponseSchema = z
+  .object({
+    output: z.record(z.unknown()).optional(),
+    message: z.record(z.unknown()).optional(),
```

Comment:
it's `output.message`

Reply:
I see `output.message` is valid only for kafka
what about `message` at root level ? Is it required then ?